.github/workflows/tests.yml

```diff
@@ -41,3 +41,5 @@ jobs:
         cd web
         npm install
         npm run lint
+        echo "Checking for clean state..."
```

Comment:
This is to catch unexpected package-lock changes right? Could we add a comment here or more clear message to ensure we remember :)

Reply:
Done.